src/UKHO.ERPFacade.Common/IO/Azure/AzureTableHelper.cs

```diff
@@ -40,22 +39,23 @@ public async Task UpsertEntity(string correlationId, string tableName, ITableEnt
             }
         }
 
-        public async Task<TableEntity> GetEntity(string correlationId, string tableName)
+        public async Task<TableEntity> GetEntity(string partitionKey, string rowKey)
```

Comment:
Async methods should all be named as such. Can you change to GetEntityAsync, and have a check for other instances where this is the case?

Reply:
Updated as suggested.